test_requirements/base.txt

```diff
@@ -1,7 +1,23 @@
 aldryn-apphook-reload>=0.2.1
+aldryn-apphooks-config
+aldryn-boilerplates
+aldryn-common>=0.1.3
+aldryn-reversion>=0.0.2
+aldryn-translation-tools
 coverage
+django-bootstrap3
+django-cms
```

Comment:
This shouldn't be in base.txt (or anywhere else). Anything that is installed in setup.py should also not be here.